install/07_servicemonitor.yaml

```diff
@@ -17,3 +17,24 @@ spec:
   selector:
     matchLabels:
       k8s-app: cluster-autoscaler-operator
+
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    k8s-app: cluster-autoscaler
+  name: cluster-autoscaler
+  namespace: openshift-machine-api
+spec:
+  endpoints:
+    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+      interval: 30s
+      port: metrics
+      scheme: http
+  namespaceSelector:
+    matchNames:
+      - openshift-machine-api
```

Comment:
This defaults to the namespace the ServiceMonitor is created in, so this is unnecessary.